src/main/java/ui/windows/StatsDelegator.java

```diff
@@ -26,6 +27,10 @@ public StatsDelegator(StatsController controller) {
     public void actionPerformed(ActionEvent e) {
         JTextField inputNameField = (JTextField)e.getSource();
         String username = inputNameField.getText();
-        controller.tryRequestUser(username);
+        try {
+            controller.tryRequestUser(username);
+        } catch (IOException ex) {
+            ex.printStackTrace();
```

Comment:
I don't think this would be the best way to handle the exception, if it is needed at all.